<launch>
  <node name="usb_cam1" pkg="usb_cam" type="usb_cam_node" output="screen" >
    <!--rosparam file = "/home/coco/jjh_cam_ws/src/calibration/calib2/calib.yaml"/-->
    <param name="angleMessage" value="angle_topic1" />
    <param name="fileName" value="/home/vialab/Sonata_ws/cam1_output.txt" />   
    <param name="video_device" value="/dev/video0" />
    <param name="image_width" value="1280" />
    <param name="image_height" value="720" />
    <param name="pixel_format" value="yuyv" />
    <param name="camera_frame_id" value="usb_cam" />
    <param name="io_method" value="mmap"/>
  </node>

  <node name="image_view" pkg="image_view" type="image_view" respawn="false" output="screen">
    <remap from="image" to="/usb_cam1/image_raw"/>
    <param name="autosize" value="false" />

  </node>


  <!--node name = "rviz" pkg = "rviz" type = "rviz" args = "-d $(find usb_cam)/rviz/robot.rviz"/-->

</launch>
